docs/ref/views.txt

```diff
@@ -93,6 +93,12 @@ Three things to note about 404 views:
     second parameter, the exception that triggered the error. A useful
     representation of the exception is also passed in the template context.
 
+.. versionchanged:: 1.10
+
+    If a non existing template name is passed, a ``TemplateNotFoundError``
+    will be raised in 2.0. A ``PendingDeprecationWarning`` is emitted
+    until then.
+
```

Comment:
This will also need to be rephrased to avoid referring to the actual warning class. I think the following text would be appropriate:

```Passing a non existing template name is deprecated and will raise a ``TemplateNotFoundError`` in Django 2.0.```